src/posix/platform/hdlc_interface.cpp

```diff
@@ -600,6 +600,15 @@ int HdlcInterface::OpenFile(const Url::Url &aRadioUrl)
         {
             tios.c_cflag |= CRTSCTS;
         }
+        else
+        {
+            tios.c_cflag &= ~(CRTSCTS);
+
+            // Deassert DTR and RTS
+            int flags;
```

Comment:
Can you please move `int flags` variable declaration to top the scope (per [style guide](https://github.com/openthread/openthread/blob/821f2415e0fc4f3ebf74e3d66f890dacf5b318b9/STYLE_GUIDE.md#white-space))

```suggestion
        else
        {
            int flags;
            
            tios.c_cflag &= ~(CRTSCTS);

            // Deassert DTR and RTS
```

Reply:
thanks for the review. Sure will fix.